client/src/components/Ticketing/ticketingmanager/DoorList/DoorList.tsx

```diff
@@ -115,6 +83,38 @@ const DoorList = (): ReactElement => {
   const [eventListFull, setEventListFull] = useState([]);
   const [eventListActive, setEventListActive] = useState([]);
   const [filterSetting, setFilterSetting] = useState('active');
+  const {token} = useFetchToken();
+
+    /**
+     * columns uses first name, last name, # of tickets purchased, arrival status, vip, donorbadge, accommodations
+     */
+    const columns = [
+        {
+            field: 'arrived',
+            headerName: 'Arrived',
+            width: 60,
+            renderCell: (params: GridCellParams) => <RenderCheckin params={params} token={token}/>,
+        },
+        {field: 'firstName', headerName: 'First Name', width: 120},
+        {field: 'lastName', headerName: 'Last Name', width: 120},
+        {
+            field: 'num_tickets',
+            headerName: 'Tickets',
+            width: 220,
+            renderCell: renderTicketTypes,
+        },
+        {field: 'email', headerName: 'Email', width: 200},
+        {field: 'phone', headerName: 'Phone Number', width: 130},
+        {field: 'vip', headerName: 'VIP', width: 65, renderCell: renderCheckbox},
+        {
+            field: 'donorBadge',
+            headerName: 'Donor',
+            width: 65,
+            renderCell: renderCheckbox,
+        },
```

Comment:
For both of these, I think it would probably make sense to just make them have a `type: boolean`, rather than a `renderCell: renderCheckbox` since it doesn't need to be an input, just a display

Reply:
Fixed